pennylane/tape/qnode.py

```diff
@@ -249,21 +252,39 @@ def _validate_backprop_method(device, interface):
         # determine if the device supports backpropagation
         backprop_interface = device.capabilities().get("passthru_interface", None)
 
+        # determine if the device has any child devices that support backpropagation
+        backprop_devices = device.capabilities().get("passthru_devices", None)
+
         if getattr(device, "cache", 0):
             raise qml.QuantumFunctionError(
                 "Device caching is incompatible with the backprop diff_method"
             )
 
         if backprop_interface is not None:
+            # device supports backpropagation natively
 
             if interface == backprop_interface:
-                return JacobianTape, None, "backprop"
+                return JacobianTape, interface, "backprop", device
 
             raise qml.QuantumFunctionError(
                 f"Device {device.short_name} only supports diff_method='backprop' when using the "
                 f"{backprop_interface} interface."
             )
 
+        if getattr(device, "analytic", False) and backprop_devices is not None:
+            # device is analytic and has child devices that support backpropagation natively
+
+            if interface in backprop_devices:
+                # TODO: need a better way of passing existing device init options
+                # to a new device?
+                device = qml.device(backprop_devices[interface], wires=device.wires, analytic=True)

```

Comment:
(minor / idea) Would it perhaps be better to simply change `self.device` here directly instead of returning the device, storing the current device as default at init (making a comment that tapes may change devices; thus there's also an `original_device`)? Not sure if that's actually an improvement though.

```suggestion
                self.device = qml.device(backprop_devices[interface], wires=device.wires, analytic=True)

```

Reply:
note that these validation methods are all 'pure' static methods - they do not access or change the state :) This is super useful for testing the logic in the unit tests

Reply:
I see. Makes sense to keep this then. :slightly_smiling_face: